terragrunt/aws/lambda.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_lambda_function" "web" {
+  function_name = "next-js-website"
+  role          = aws_iam_role.web.arn
+  package_type  = "Image"
+  memory_size   = 1024
+  timeout       = 30
+  image_uri     = "${aws_ecr_repository.web.repository_url}:latest"
+
+  environment {
```

Comment:
Just as a caution here - these environment variables are always plain text and can leak if you put secrets.  Totally doesn't apply in your case, but figured I'd save you the pain I went through with the leak we had a few years ago 😅